http/media/jsonp/src/main/java/io/helidon/http/media/jsonp/JsonpSupport.java

```diff
@@ -90,8 +99,31 @@ public static MediaSupport create(Config config) {
     public static MediaSupport create(Config config, String name) {
         Objects.requireNonNull(config);
         Objects.requireNonNull(name);
+        return builder().name(name).build();
+    }
+
+    /**
+     * Creates a new {@link JsonpSupport} based on the {@link JsonpSupportConfig}.
+     *
+     * @param jsonpSupportConfig must not be {@code null}
+     * @return a new {@link JsonpSupport}
+     */
+    public static JsonpSupport create(JsonpSupportConfig jsonpSupportConfig) {
+        Objects.requireNonNull(jsonpSupportConfig);
+        return new JsonpSupport(jsonpSupportConfig);
+    }
+
+    /**
+     * Creates a new builder.
+     *
+     * @return a new builder instance
+     */
+    public static JsonpSupportConfig.Builder builder() {
+        return JsonpSupportConfig.builder();
+    }
 
-        return new JsonpSupport(name);
+    static JsonpSupport create(Consumer<JsonpSupportConfig.Builder> consumer) {
```

Comment:
why is this method not public?

Reply:
Oh. Forgot to add public. Will fix that.